src/routes/(app)/Navbar.svelte

```diff
@@ -7,7 +7,13 @@
   import { routes } from "./routes";
   import { isAuthorized } from "$lib/utils/authorization";
   import { getFullName } from "$lib/utils/client/member";
-  import Notification from "./NotificationBell.svelte";
+  import NotificationBell from "./NotificationBell.svelte";
+  import type { Notification } from "@prisma/client";
+  import type { SuperValidated } from "sveltekit-superforms";
+  import type { NotificationSchema } from "$lib/zod/schemas";
+
+  export let notifications: Notification[] | null;
```

Comment:
Maybe we should get notifications from $page.data.notifications instead? I'm not sure what's best.

Reply:
I agree